### PR TITLE
minor type fixes

### PR DESCRIPTION
**Pull request by jonath92**
just some minor fixes/additions which help me for the radio applet (most I figured out recently when changing to strict mode)

### CODE REVIEW

File: XApp.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+declare namespace imports.gi.XApp {
```
Comment (Gr3q) [inline XApp.d.ts]: Can you remove this file? I added the generation from it's .gir file for it for you

**Review by Gr3q — CHANGES_REQUESTED** [review]

File: XApp.d.ts (hunk 1 above)
Comment (jonath92) [reply XApp.d.ts]: Done. That's obviously much better :-)

File: index.d.ts [hunk 2]
```diff
@@ -60,5 +60,6 @@
 /// <reference path="./cinnamon/ui/xdndHandler.d.ts" />
 
 /// <reference path="./cinnamon/globals.d.ts" />
+/// <reference path="./XApp.d.ts" />
```
Comment (Gr3q) [inline index.d.ts]: This line as well because it doesn't exist anymore please

**Review by Gr3q — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 2 above)
Comment (jonath92) [reply index.d.ts]: Sorry forgot to delete. Now deleted.

**Review by Gr3q — APPROVED** [review]
Thanks, I'll create the new release in the evening

**Pull request merged**